### PR TITLE
Devise installed and users, games and bookings migrated

### PR DESCRIPTION
**Pull request by soojinhwang**
Managed to install devise to create the user model, and also re-created the game and booking models
Will move onto adding back the validations and setting up the controllers

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -13,6 +13,9 @@
   </head>
 
   <body>
+    <p class="notice"><%= notice %></p>
```
Comment (Aisselaa) [inline app/views/layouts/application.html.erb:16]: Checked while re-watching Sat's lecture. Looks good!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -13,6 +13,9 @@
...
+    <p class="alert"><%= alert %></p>
+
```
Comment (Aisselaa) [inline app/views/layouts/application.html.erb:18]: checked while re-watching Sat's lecture. Looks good!

Comment (soojinhwang) [reply app/views/layouts/application.html.erb:18]: Cool, think that's been added now, I'll commit again

File: app/models/game.rb [hunk 3]
```diff
@@ -1,19 +1,3 @@
 class Game < ApplicationRecord
   belongs_to :user
```
Comment (SamuelGebru) [inline app/models/game.rb:2]: Hi Soo, does this line not need the  .... has_many :bookings, dependent: :destroy

Comment (SamuelGebru) [reply app/models/game.rb:2]: Never mind, it has it on the other commits..

**Review by SamuelGebru — APPROVED** [review]
This look great!

**Comment by mfjoe** [comment]
Good job soo

**Pull request merged**